### ISSUE TITLE
Sound problem

### ISSUE THREAD

**Issue by skelletR** [nikita36078/J2ME-Loader#136]
Hi when I get out  of the J2ME-Loader and go in home screen or another app without close the J2ME-Loader app when back again the sound not working and must close the app and open again to fix

**Comment by oatmeal01** [comment]
All games installed?

**Comment by skelletR** [comment]
Yes sometimes now always.. Now I try it and no problems only some times just say it maybe it a issue or just in my phone

**Comment by jasper1412** [comment]
Similar problem. Returning to J2ME-Loader menu after exiting a game, for example Rally 3D, midi still plays even though it was closed. I need to go force closing J2ME-Loader to stop those midi playing. This also happen sometimes on original java-enabled mobile phones, so I think this was really a problem on J2ME framework itself and not the emulator as it only interprets the original. But I hope this would fixed as soon as possible :)

**Comment by skelletR** [comment]
Yes exactly

**Comment by oatmeal01** [comment]
Is this fixed? I have an issue about Shado Fighter too and the sound still plays after exiting the game. The latest update 1.2.7.7 fixed the issue though.

**Comment by jasper1412** [comment]
nope, I think 1.2.7.7 only fixes it on some games but not all, Rally 3D still having those issue.

**Comment by oatmeal01** [comment]
Downloaded rally 3d and experienced the same issue too. The game  is also laggy

**Comment by jasper1412** [comment]
Sound is the only issue I have on Rally 3D. I notice lags on other games I've already tested on previous builds, every release makes it slower and slower, but I wouldn't mind reporting those as those optimization will come sometime in development. You may try testing **Asphalt Nitro** game `480x800`, one of the levels have their **SMOOTHEST** gameplay performance.

**Comment by oatmeal01** [comment]
what version of rally 3d do you have, ill try a different version and ill try nitro too

**Comment by oatmeal01** [comment]
what version of rally 3d do you have, ill try a different version and ill try nitro too

**Comment by jasper1412** [comment]
I think there are only 2 versions of Rally 3D, mine's `240x320`.

**Comment by oatmeal01** [comment]
My version of rally 3d is 1.2 by nokia, tried rally pro contest 3d by synergix but the graphics is messy

**Comment by jasper1412** [comment]
You tripped on 3d limitation, you may want to report it as separate issue to be reviewed later.

**Comment by oatmeal01** [comment]
I've downloaded the right version, 240x320 v1.2 s40 edition

**Comment by oatmeal01** [comment]
Rally 3d works fine now fps is smooth not like before, i think my old downloaded version is for s60

**Comment by jasper1412** [comment]
this is fixed on 1.2.7.8
